Pod/Sources/OriginateRemoteCollection+Internal.h

```diff
@@ -0,0 +1,33 @@
+//
+//  OriginateRemoteCollection+Internal.h
+//  Pods
+//
+//  Created by Philip Kluz on 10/3/15.
+//
+//
+
+@import Foundation;
+
+typedef NS_ENUM(NSUInteger, OriginateRemoteCollectionState) {
+    OriginateRemoteCollectionStateNone,
+    OriginateRemoteCollectionStateIdle,
+    OriginateRemoteCollectionStateLoading
+};
+
+@interface OriginateRemoteCollection ()
+{
+    NSArray *_objects;
+}
+
+#pragma mark - Properties
+@property (nonatomic, assign, readwrite) OriginateRemoteCollectionState state;
+@property (nonatomic, strong, readwrite) NSError *lastError;
+@property (nonatomic, strong, readwrite) NSArray *objects;
```

Comment:
unpossible without introducing a second `ivar` just for the mutable case. Methods inside the superclass, that are called and refer to `_objects` would access their version instead of dynamic dispatch to the correct classes ivar